doc/schemas/bkpr-listincome.request.json

```
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "added": "pre-v0.10.1",
+  "required": [],
+  "additionalProperties": false,
+  "properties": {
+    "start_time": {
+      "type": "u32",
+      "description": "Filters the events after the time"
+    },
+    "debit_time": {
+      "type": "u32",
+      "description": "Filters the events upto the timestamp"
+    },
+    "consolidate_fee": {
```

```suggestion
    "consolidate_fees": {
```

For this reason the CI is failing

---

Ah sorry for this.